### PR TITLE
Support for JsonSchema in Kafka Read Schema Transform

### PR DESCRIPTION
**Pull request by pabloem**
This PR depends on https://github.com/apache/beam/pull/24271

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -678,6 +680,8 @@ class BeamModulePlugin implements Plugin<Project> {
         joda_time                                   : "joda-time:joda-time:2.10.10",
         jsonassert                                  : "org.skyscreamer:jsonassert:1.5.0",
         jsr305                                      : "com.google.code.findbugs:jsr305:$jsr305_version",
+        json_org                                    : "org.json:json:${json_org_version}",
```
Comment (Abacn) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:683]: Thanks for the change! We already have a couple of json parser dependencies (e.g. jackson, gson, fastjson) throughout beam project. And there is some voice to reduce them (#24154). Could one (or some) of these libs satisfy the need of functionality here, or it is necessary to introduce these new deps?

Comment (pabloem) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:683]: thanks for the observation @Abacn - let's keep this discussion in https://github.com/apache/beam/pull/24271 where the code is added? : )

Comment (pabloem) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:683]: this PR is blocked by https://github.com/apache/beam/pull/24271

**Comment by damccorm** [comment]
stop reviewer notifications

**Comment by pabloem** [comment]
Run Java PreCommit

**Comment by pabloem** [comment]
r: @ahmedabu98 : )

**Comment by pabloem** [comment]
Run Java PreCommit

**Review by ahmedabu98 — APPROVED** [review]
Thanks Pablo! I appreciate the Class name changes :) LGTM

**Pull request merged**